src/scripts/main.js

```diff
@@ -1 +1,84 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+function creatDiv(str, className) {
```

Comment:
There is a typo in the function name `creatDiv`. It should be `createDiv` to reflect the correct spelling.